spec.md

```diff
@@ -880,9 +880,10 @@ message ControllerServiceCapability {
 
 #### `NodePublishVolume`
 
-A Node Plugin MUST implement this RPC call.
-This RPC is typically called by the CO when it wants to place a workload that wants to use the volume on a node.
+This RPC is called by the CO when a workload that wants to use the specified volume is placed (scheduled) on a node.
 The Plugin SHALL assume that this RPC will be executed on the node where the volume will be used.
+This RPC may be called by the CO multiple times: at least once per volume per workload
```

Comment:
Hi @saad-ali, the "workload" might mean different things on different COs. For instance, it might be container for Docker, and pod for Kubernetes. For Mesos, I don't even know what "workload" will mean because it supports both pod style and container style workload. If we want to call out "workload", we must clearly define what is that in the spec.

I think the motivation for this change is to allow the CO to publish a volume on a node in different ways, instead of just one way. For instance, mount a fuse filesystem with different  options. I think if we say in the spec that the CO MAY call this multiple times, then that should be suffice to force the SP to be written in such a way that it can handles multiple calls to `NodePublishVolume`.

We should call out different `target_path` I think. Because it's super weird to me that the CO call `NodePublishVolume` multiple times with the same `target_path` but with different parameters.